clients/admin-ui/src/home/HomeContent.tsx

```diff
@@ -0,0 +1,119 @@
+import { Center, Flex, SimpleGrid, Text } from "@fidesui/react";
+import Link from "next/link";
+import * as React from "react";
+import { useEffect, useState } from "react";
+
+import { useFeatures } from "~/features/common/features.slice";
+
+import { MODULE_CARD_ITEMS, ModuleCardKeys } from "./constants";
+import { ModuleCard } from "./types";
+
+const HomeContent: React.FC = () => {
+  const COLUMNS = 3;
+  const [list, setList] = useState<ModuleCard[]>(
+    MODULE_CARD_ITEMS.filter((item) =>
+      [
+        ModuleCardKeys.ADD_SYSTEMS,
+        ModuleCardKeys.CONFIGURE_PRIVACY_REQUESTS,
+      ].includes(item.key)
+    )
+  );
+  const { connectionsCount, systemsCount } = useFeatures();
+  const hasConnections = connectionsCount > 0;
+  const hasSystems = systemsCount > 0;
+
+  const getCardItem = (key: ModuleCardKeys) => {
+    const cardItem = MODULE_CARD_ITEMS.find((item) => item.key === key);
+    return cardItem;
+  };
+
+  useEffect(() => {
+    if (hasConnections || hasSystems) {
+      if (hasConnections) {
+        const card = getCardItem(ModuleCardKeys.REVIEW_PRIVACY_REQUESTS);
+        if (card && !list.includes(card)) {
+          setList((prev) => [...prev, card as ModuleCard]);
```

Comment:
I was wondering why this `as` was necessary. Looks like the `getCardItem` can [return](https://github.com/ethyca/fides/blob/6de0ba227ae789c9f0b485491a10e924712d3cfc/clients/admin-ui/src/home/HomeContent.tsx#L26) `undefined` because that's what `find` returns. However, that function is takes a `ModuleCardKeys` as its argument, so it should be impossible for the find to fail. 

So, that function should be responsible for the exclamation-mark-assert-defined to remove that undefined case.

Or that function can raise an exception because that case would only be possible under mis-configuration.

Or, to avoid the undefined case entirely, the constants file could export `KEY_TO_MODULE_CARD_ITEMS` object whose keys are typed as `ModuleCardKeys` and whose values are are the things in `MODULE_CARD_ITEMS`. Then if we were to go and mis-configure the module cards, typescript would yell at us that a property is missing.

Reply:
Issue has been addressed.